src/obfl-specification.html

```diff
@@ -1574,13 +1577,34 @@ <h4 id="L1105" class="include">toc-entry</h4>
         <dt>ref-id [required]</dt>
           <dd>The id of the block this entry is connected with.</dd>
       </dl>
+    </dd>
+  <dt>Content Model</dt>
+    <dd>toc-entry-content [1 or more], toc-entry [0 or more]</dd>
+	<dd>There must be exactly one toc-entry-content element without a use-when attribute.</dd>
```

Comment:
The limitation "There must be exactly one toc-entry-content element without a use-when attribute" is not really needed I think. You should be able to say e.g. `use-when="true"`, or even if none of the `toc-entry-content` match, Dotify should be able to cope with that by just not inserting an entry.

In fact this limitation could even be problematic because you wouldn't be able to have a single `toc-entry-content` with `use-when="(not $resumed)"`, which might be needed to get the current behavior (depending on what the default value of `use-when` is).

I think you could even argue changing the content model to "toc-entry-content [0 or more], toc-entry [0 or more]". The way it is now, Pipeline will need to generate empty `toc-entry-content` in some cases where a `toc-entry` is used merely as a container for other `toc-entry`s. This is not a problem, but it does show that a `toc-entry-content` is not always needed.